addons/tooltips/addon.json

```diff
@@ -0,0 +1,112 @@
+{
+  "name": "Tooltips",
+  "description": "Adds tooltips for projects, users, studios, forum topics, and forum posts. Uses the Scratch API and ScratchDB.",
+  "credits": [
+    {
+      "name": "Hans5958"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "script.js",
+      "matches": "*"
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "base.css",
+      "matches": "*"
+    },
+    {
+      "url": "scratch2.css",
+      "matches": "*",
+      "if": {
+        "settings": {
+          "theme": "scratch2"
+        }
+      }
+    },
+    {
+      "url": "scratch3.css",
+      "matches": "*",
+      "if": {
+        "settings": {
+          "theme": "scratch3"
+        }
+      }
+    },
+    {
+      "url": "extended.css",
+      "matches": "*",
+      "if": {
+        "settings": {
+          "theme": "extended"
+        }
+      }
+    },
+    {
+      "url": "extended-statistics.css",
+      "matches": "*",
+      "if": {
+        "settings": {
+          "theme": "extendedStatistics"
+        }
+      }
+    }
+  ],
+  "settings": [
+    {
+      "id": "theme",
+      "name": "Theme",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "scratch3",
+          "name": "Scratch 3"
+        },
+        {
+          "id": "scratch2",
+          "name": "Scratch 2"
+        },
+        {
+          "id": "extended",
+          "name": "Extended"
+        },
+        {
+          "id": "extendedStatistics",
+          "name": "Extended (statistics)"
+        }
+      ],
+      "default": "scratch3"
+    },
+    {
+      "id": "color-scheme",
+      "name": "Color scheme",
+      "type": "select",
+      "potentialValues": [
+        {
+          "id": "auto",
+          "name": "Auto"
+        },
+        {
+          "id": "light",
+          "name": "Light"
+        },
+        {
+          "id": "dark",
+          "name": "Dark"
+        }
+      ],
+      "default": "auto"
+    },
```

Comment:
Is it worth it to implement a theme for this separate from `dark-www`? I can't imagine that many users would want to set this to "light" when they have dark mode enabled, and vice versa.

Reply:
Well, for me personally, I wanted to do light when I have dark mode. I'm sure it's not just me

Reply:
Have you tried increasing the brightness of the background of the dark tooltip?

Reply:
Do it too contrast and it won't fit with the rest of the page. So, yeah.

Reply:
Oh, yeah, it's possible to have dark tooltip on light. It is also nice.

Reply:
Customizability is a good thing, but I think it's better if the settings for an addon that isn't a theme control its functionality and not the appearance. A border similar to that of the emoji picker could be added to make the separation from the background more obvious in dark mode.

Reply:
I would disagree on this. Some people have different preferences on this. Tippy's default is black, while GitHub Hovercards have white background.

I'd rather expand this future to something like "text color" and "background color". Maybe a global setting on that plugin would be better, but there should be a default nonetheless.

Reply:
Yes, different designs exists, but I think it's better to choose a single good design and focus on the functionality of the addon instead. If the option to customize this is implemented anyway, emoji picker should have it as well.